docs/user/reporting/index.asciidoc

```diff
@@ -14,13 +16,55 @@ Reporting is located in the share menu from the {kib} toolbar:
 [role="screenshot"]
 image::user/reporting/images/share-button.png["Share"]
 
-You can also <<automating-report-generation, generate reports automatically>>.
+[float]
+== System setup
+
+{reporting} is automatically enabled in {kib}. The first time {kib} runs, it extracts a custom build for the Chromium web browser, which
+runs on the server in headless mode to load {kib} and capture the rendered {kib} charts as images.
+
+Chromium is an open-source project not related to Elastic, but the Chromium binary for {kib} has been custom-built by Elastic to ensure it
+works with minimal setup. However, the {kib} server OS might still require additional dependencies for Chromium. See the
+<<reporting-troubleshooting-system-dependencies, Reporting Troubleshooting>> section for more information about the system dependencies
+for different operating systems.
+
+[float]
+== Manually generate reports
+
+. Open {kib} in your web browser and log in. If you are running {kib}
+locally, go to `http://localhost:5601`. To access {kib} and generate
+reports, you need the `kibana_user` and `reporting_user` roles. For more
+information, see <<secure-reporting>>.
+
+. Open the dashboard, visualization, or saved search you want to include
+in the report.
+
+. Click *Share* in the {kib} toolbar:
++
+[role="screenshot"]
+image:user/reporting/images/share-button.png["Reporting Button",link="share-button.png"]
+
+. Depending on the {kib} application, choose the appropriate options:
```

Comment:
Can you make this so it isn't a three level list?  Instead

4.  If you're on Discover, select CSV Reports and click Generate CSV. 

5.  If you're on Visualize or Dashboard
      a.
      b.   
      c.